app/news/[id]/page.tsx

```
@@ -0,0 +1,29 @@
+import { getPostById } from "@/lib/posts";
+import Link from "next/link";
+import "../markdown.css";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)